reminders/migrations/0002_add_reminder_test_data.py

```diff
@@ -0,0 +1,46 @@
+from django.db import migrations, transaction
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('reminders', '0001_initial'),
+        ('users', '0003_createsuperuser'),
+        ('events', '0004_add_event_participant_test_data')
+    ]
+
+    def generate_reminder_test_data(apps, schema_editor):
+        from reminders.models import Reminder
+        from events.models import EventParticipant
+        from django.utils import timezone
+        from datetime import datetime
+
+        reminder_data = [
+            (EventParticipant.objects.get(event_id__title="event1", user_id__username="testUser1"),
+             datetime(2022, 4, 24, 11, 11, 11, 0, tzinfo=timezone.utc),
+             'Event in {} hours'.format(1),
+             Reminder.EMethods.WEBSITE_EMAIL,
```

Comment:
as I suggested below if you take the ReminderType outside of the Reminder class it should looks like:
```suggestion
             ReminderType.WEBSITE_EMAIL,
```